package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "playwright-core",
-  "version": "1.0.0-post",
+  "version": "0.15.0-post",
```

Comment:
that seems not intended

Reply:
Rather 1.0.0 was not intended, it was too early :)